packages/core/core/src/requests/ParcelConfigRequest.js

```diff
@@ -172,6 +172,17 @@ export async function resolveParcelConfig(
     contents,
     options,
   );
+
+  if (options.additionalReporters.length > 0) {
```

Comment:
Adding reporters shouldn't invalidate the cache. Can you make sure this option isn't tracked in the graph? I believe there is a list of ignored options.